modules/backend/constants/queryOptions.ts

```diff
@@ -115,27 +113,24 @@ export const verifiedEmailsInfiniteOptions = (options?: PageSize) => {
   })
 }
 
-export const ticketPriceOptions = (
-  body: GetTicketPriceRequestDto,
-  {
-    udr,
-    licencePlate,
-    duration,
-    npk,
-  }: {
-    udr: MapUdrZone | null
-    licencePlate: string
-    duration: number
-    npk: ParkingCardDto | null
-  },
-) =>
+export const ticketPriceOptions = (body: GetTicketPriceRequestDto) =>
   queryOptions({
-    queryKey: ['TicketPrice', udr, licencePlate, duration, npk],
+    queryKey: [
+      'TicketPrice',
+      body.ticket.udr,
+      body.ticket.ecv,
+      body.ticket.parkingEnd,
+      body.npkId,
+      body.ticket.udrUuid,
+      body.ticket.parkingStart,
+    ],
     queryFn: () => clientApi.ticketsControllerGetTicketPrice(body),
     select: (res) => res.data,
     // https://tanstack.com/query/latest/docs/react/guides/migrating-to-v5#removed-keeppreviousdata-in-favor-of-placeholderdata-identity-function
     placeholderData: keepPreviousData,
-    enabled: !!udr && !!licencePlate && !!duration,
+    refetchInterval: 1000 * 60,
+    refetchIntervalInBackground: true,
```

Comment:
Do we really want to refetch it in background? Isn't use `useQueryWithFocusRefetch` enough? Can refetching in background cause significant increase in number of requests?